### ISSUE TITLE
Wrong path when file specified as args

### ISSUE THREAD

**Issue by weijiangan** [securego/gosec#210]
### Summary 
Unable to check file when filename is specified as args because `gas` doesn't get the correct file path. It repeats the repo name twice and results in an invalid path

### Steps to reproduce the behavior
Run `gas` command with filename as args, i.e.

```sh
$ gas -fmt=csv main.go
[gas] 2018/05/24 03:22:34 including rules: default
[gas] 2018/05/24 03:22:34 excluding rules: default
[gas] 2018/05/24 03:22:34 Skipping: /Users/weijiangan/go/src/somegit.service/user/git-repo/somegit.service/user/git-repo/main.go. Path doesn't exist.
[gas] 2018/05/24 03:22:34 no initial packages were loaded
```

### Gas version
5/24/2018

### Go version (output of 'go version')
go1.10.1 darwin/amd64

### Operating system / Environment
macOS 10.13.5

### Expected behavior
Check file in args

### Actual behavior
File path processing botched

**Comment by gcmurphy** [comment]
Can you please provide the value of your $GOPATH?

**Comment by weijiangan** [comment]
Here's the output of git rev-parse HEAD:
```
4ae8c95b4097691ea5b2b0c5060f4ca0413dfcef
```
my `$GOPATH` is `/Users/weijiangan/go`

Thanks!

**Comment by ccojocar** [comment]
Fixed by #217
